doc/Programming-guide/Key-platform-features/Endpoint-profiles/index.md

```diff
@@ -300,20 +309,19 @@ profile->destroy(profile);
 </div>
 </div>
 
-## Server-side endpoint profile ##
+## Server-side endpoint profile
 
-The server-side endpoint profile is initially generated at the stage of a new endpoint registration. By default, server side profile record is auto-generated based on 
-the latest server-side profile schema of particular application. 
+The server-side endpoint profile is initially generated at the stage of a new endpoint registration.
+By default, server-side profile record is auto-generated based on the latest server-side profile schema of a particular application.
 In this case, endpoint membership in the endpoint groups is re-evaluated and updated to match the new endpoint profile.
```

Comment:
Why is it uniquely in this case? Membership is re-evaluated on any profile change.